cms/static/js/certificates/views/signatory_editor.js

```diff
@@ -105,7 +109,8 @@ function ($, _, Backbone, gettext,
             if (event && event.preventDefault) { event.preventDefault(); }
             this.model.set(
                 'organization',
-                this.$('.signatory-organization-input').val()
+                this.$('.signatory-organization-input').val(),
+                { silent: true }
```

Comment:
What is the reason for not calling `this.toggleValidationErrorMessage('orgnaization');` here?